game.js

```diff
@@ -4,6 +4,7 @@ class Game {
         this.player2ID = 'O';
         this.currentPlayer = this.player1ID;
         this.board = new Board();
+        this.askedQuestions = [];
```

Comment:
I definitely think this *should* belong in the Game class, but I think that we should put it in the `window.addEventListener...` code so that we don't have to make adjustments to all of the diagrams and the SRS document